cs/src/Contracts/TunnelEndpoint.cs

```diff
@@ -61,6 +61,13 @@ public abstract class TunnelEndpoint
     [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)]
     public string? PortUriFormat { get; set; }
 
+    /// <summary>
+    /// Gets or sets the default URIs without the port information in the url
+    /// and the web forwarding forwards requests to this url to default port.
```

Comment:
The phrasing of this comment is  confusing. How about this?

`Gets or sets the URI where a web client can connect to the default port of the tunnel.`